src/components/conditionals/JoinButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { reserveMissions } from '../../Redux/State/MissionsState';
+
+const JoinButton = (missionsObj) => {
+  const { id } = missionsObj;
+  const dispatch = useDispatch();
+  const handleClick = (e) => {
+    console.log('clicked', e.target.id);
+    dispatch(reserveMissions(e.target.id));
+  };
+  return (
+    <button
+      id={id}
+      type="button"
+      onClick={(e) => handleClick(e)}
+      className="joinMissionButton"
+    >
+      Join
+    </button>
+  );
```

Comment:
_<h2>Optional : </h2>_
_- Delete the console.logs_
_- Might aswell consider :  `onClick={(e) => dispatch(reserveMissions(e.target.id))}` (instead of two fucntions)_